src/browser/modules/D3Visualization/components/Inspector.jsx

```diff
@@ -39,12 +39,12 @@ import {
 import { GrassEditor } from './GrassEditor'
 import { RowExpandToggleComponent } from './RowExpandToggle'
 import ClickableUrls from '../../../components/clickable-urls'
+import numberToUSLocale from '../../../../shared/utils/number-to-US-locale'
```

Comment:
You should be able to import from `services/utils/number-to-US-locale`. Maybe we need to adjust our aliases if this doesn't work for you.

Reply:
ah Ill give it a go, didnt see that